examples/vector_similarity_search.ipynb

```diff
@@ -0,0 +1,420 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Vector Similarity Search"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates usage of the vector similarity search, to help the agent select an exact query value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')"
```

Comment:
Why do we need this? Are there warning from Langchain?

Reply:
there were, but we actually don't need this here because i can suppress the warnings in an ipynb config file